docs/configurationProvisioning.md

```diff
@@ -7,20 +7,20 @@ Configuration Provisioning guide
 * [Usage](#overview)
```

Comment:
Although this is not part of the PR, could you please update the links of the index?